testcases/MachineConfig.py

```diff
@@ -325,6 +325,18 @@ def LparSetup(self):
             ioslot_drc_names = ",".join(ioslot_drc_names[:ioslot_drc_names.index("=")].split(",")[:-1]) \
                                if "=" in ioslot_drc_names else ioslot_drc_names
             self.cv_HMC.add_ioslot(ioslot_drc_names)
+        
+        if "perf=1" in self.machine_config:
+            conf = OpTestConfiguration.conf
+            if self.cv_HMC.is_perfcollection_enabled():
+                log.info("System is already booted with perf collection profile enabled")
+            else:
+                self.cv_HMC.hmc_perfcollect_configure()
+                if self.cv_HMC.is_perfcollection_enabled:
+                    log.info("System is already booted with perf collection profile enabled")
+                else:
+                    return "Failed to boot perf collection profile enabled"
```

Comment:
return a string ? or this should be log.info ? or raise error

Reply:
+1. Also the message string should be 
"Failed to enable Performance Information collection"

Reply:
Thanks Sachin, I will reword , as we need to return statement with string as in caller we are making test fails with retun string